src/components/sidebar/sidebar.spec.tsx

```diff
@@ -25,14 +25,7 @@ describe("Sidebar", () => {
   beforeEach(() => {
     spy = jest.fn();
     wrapper = mount(
-      <Sidebar
-        open
-        title="Test"
```

Comment:
not really related to this file but I notice that although there is no `title` prop on `Sidebar` (we call it `header` instead - I think in other components `title` is a simple string which gets included in a styled header but Sidebar doesn't have it for some reason), it is referred to in the doc comments for some other props like the `aria-label`. It would be good to update that either now or later.

Reply:
Also this isn't currently visible as the prop table doesn't show up (it doesn't on master either). Not sure why as it's definitely not any of the usual reasons like not exporting the component as a named export.

Reply:
Good spot, I'll address these in this PR 👍